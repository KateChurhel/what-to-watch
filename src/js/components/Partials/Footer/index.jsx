// libraries
import React from 'react';
import { Link } from 'react-router-dom';

const Footer = () => (
  <footer className="page-footer">
    <Link className="logo logo--light" to="/">WTW</Link>

    <div className="copyright">Â© 2020 What to watch Ltd.</div>
  </footer>
);

export default Footer;
