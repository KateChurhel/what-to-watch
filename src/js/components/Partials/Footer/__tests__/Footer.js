import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Footer from '../index';

it('Footer correctly renders', () => {
  const tree = shallow(<Footer />);
  expect(toJSON(tree)).toMatchSnapshot();
});
