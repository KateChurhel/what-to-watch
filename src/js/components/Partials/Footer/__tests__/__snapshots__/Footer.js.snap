// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer correctly renders 1`] = `
<footer
  className="page-footer"
>
  <Link
    className="logo logo--light"
    to="/"
  >
    WTW
  </Link>
  <div
    className="copyright"
  >
    Â© 2020 What to watch Ltd.
  </div>
</footer>
`;
